src/modules/vtol_att_control/vtol_type.cpp

```diff
@@ -422,6 +422,43 @@ bool VtolType::is_channel_set(const int channel, const int target)
 
 float VtolType::pusher_assist()
 {
+	// Altitude above ground is distance sensor altitude if available, otherwise local z-position
+	float dist_to_ground = -_local_pos->z;
+
+	if (_local_pos->dist_bottom_valid) {
```

Comment:
@RomanBapst I assume dist_bottom_valid is only true when you have a distance sensor connected? How come then that it still has a value even without one, shouldn't it then be NAN?